catalog/rocks/cataloger.go

```diff
@@ -691,18 +703,29 @@ func newCatalogEntryFromEntry(commonPrefix bool, path string, ent *Entry) catalo
 	return catEnt
 }
 
-func newDifferenceFromEntryDiff(v *EntryDiff) catalog.Difference {
-	var diff catalog.Difference
-	switch v.Type {
+var ErrUnknownDiffType = errors.New("unknown graveler difference type")
```

Comment:
can you move this line to the start of the file - it will help to find globals and keep a file structure

Reply:
Sure.